### DISCUSSION TITLE
beforescriptexecute/afterscriptexecute event substitute

### DISCUSSION THREAD

**Discussion by KDeluxe2023** [quoid/userscripts#417] [Q&A]
I switched over to safari from firefox.
I used to have some functionality dependent on [beforescriptexecute event.](https://developer.mozilla.org/en-US/docs/Web/API/Document/beforescriptexecute_event)
Sadly webkit is never going to have it because it was removed from spec (makes you wonder why hehe).
Anyway I was wondering if there's some other way to execute code before specific js gets loaded?

I'd really appreciate it, thanks

**Comment by ACTCD** [comment]
In short, if you need to prevent or alter the script from running, I don't think there is an alternative.
